modules/repository-datastax/src/test/java/com/intuit/wasabi/repository/cassandra/impl/CassandraExperimentRepositoryTest.java

```diff
@@ -722,4 +723,31 @@ public void testGetTagList() throws ExecutionException, InterruptedException {
         assertArrayEquals(exp2.toArray(), callResult.get(app02).toArray());
     }
 
+
+    @Test
+    public void testGetEmptyTagList() throws ExecutionException, InterruptedException {
+        //------ Input --------
+        Application.Name app01 = Application.Name.valueOf("app01");
```

Comment:
It improves readability of the code in general and it becomes easier for anybody else to understand later in case any modifications or debugging needs to be done later.